collector/rabbitmq/config.yaml

```diff
@@ -0,0 +1,26 @@
+receivers:
+  prometheus:
+  prometheus/rabbitmq:
+    buffer_period: 234
+    buffer_count: 45
```

Comment:
are these parameters we can leave out? They look like options the user might not care about--I assume they're general to the Prometheus receiver.